dynamiqs/integrators/apis/jsmesolve.py

```diff
@@ -64,14 +65,9 @@ def jsmesolve(
         - $L_k\to L_k(t)$
         - $\dd N_k\to \dd N_k(t)$
 
-    Note-: Jump vs. diffusive SME
-        In quantum optics the _jump_ SME corresponds to photon counting schemes, as
-        opposed to the _diffusive_ SME which corresponds to homodyne or heterodyne
-        detection schemes.
-
     The continuous-time measurements are defined by the point processes $\dd N_k$. The
-    solver returns the times at which the detector clicked:
-    $I_k = \{t \text{ s.t. } \dd N_k(t)=1\}$.
+    solver returns the times at which the detector clicked,
+    $I_k = \{t \in [t_0, t_\text{end}[ \,|\, \dd N_k(t)=1\}$.
```

Comment:
`t_0`  and `t_\text{end}` are not defined.

Reply:
Sure, but it's quite obvious, no? I don't think we need to be 100% rigorous everywhere.

Reply:
Yes ok 👍